### PR TITLE
ISORT: Add _typing to pre_core isort block

### LINKED ISSUE #27534: STY: add pandas._typing to the known_pre_core block for isort

### PR DESCRIPTION
**Pull request by alimcmaster1**
- [x] closes #27534

### CODE REVIEW

File: asv_bench/benchmarks/io/excel.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 from io import BytesIO
 
 import numpy as np
+
 from odf.opendocument import OpenDocumentSpreadsheet
 from odf.table import Table, TableCell, TableRow
 from odf.text import P
-
```
Comment (jbrockmendel) [inline asv_bench/benchmarks/io/excel.py]: Isort complaint here

**Comment by jreback** [comment]
can you rebase and ping on green. as just merged some more things.

**Comment by alimcmaster1** [comment]
@jreback done and green - thanks!

**Comment by jreback** [comment]
thanks @alimcmaster1 

is this automatically pickuped up in the pre-commit hooks? (I think yes, just confirming)

**Pull request merged**